packages/lit-analyzer/src/analyze/lit-analyzer-config.ts

```diff
@@ -165,6 +168,7 @@ export function makeConfig(userOptions: Partial<LitAnalyzerConfig> = {}): LitAna
 	return {
 		strict: userOptions.strict || false,
 		rules: makeRules(userOptions),
+		securitySystem: userOptions.securitySystem || "off",
```

Comment:
Does this come straight from a user config file, or can we assume it's validated as a partial config already? (Otherwise should this validate and throw?)

Reply:
Following convention here, I didn't throw. Added some validation, since I didn't see other validation. @runem what do you think?

Reply:
The configuration given to `lit-analyzer` is not validated as of now. First of all, I think the CLI should throw on invalid values, but `ts-lit-plugin` should only log warnings (no reason to crash when running in the IDE). 

Therefore, I think that we should implement one of these solutions:
a. export a `validateConfig` function that validates the raw, partial config.
b. extend `makeConfig` with an option to assert values (perhaps a callback that is called on invalid config values).

I also think `makeConfig` should properly fall back to default values when encountering an invalid value (which it doesn't do for all attributes right now). 

I created an issue for it here https://github.com/runem/lit-analyzer/issues/65 so that it can be done in a future, separate PR 👍